src/components/panningViews/panDismissibleView.js

```diff
@@ -29,13 +27,14 @@ class PanToDismissView extends BaseComponent {
      */
     onDismiss: PropTypes.func,
     /**
-     * The animation speed (default is 20)
+     * Some animation options to choose from:
+     * animationSpeed - The animation speed (default is 20)
+     * animationBounciness - The dismiss animation duration (default is 280)
      */
-    animationSpeed: PropTypes.number,
-    /**
-     * The animation bounciness (default is 6)
-     */
-    animationBounciness: PropTypes.number,
+    animationOptions: PropTypes.shape({
+      animationSpeed: PropTypes.number,
```

Comment:
now that we have `animationOptions` no need for the `animation` prefix for each inner prop (speed, bounciness are enough)

Reply:
Done